test/cbot/CMakeLists.txt

```diff
@@ -1,19 +1,13 @@
-# Includes
-include_directories(
-    ${COLOBOT_LOCAL_INCLUDES}
+add_executable(CBot_console console.cpp)
+target_link_directories(CBot_console PRIVATE
     ${CMAKE_CURRENT_SOURCE_DIR}
     ${CMAKE_CURRENT_BINARY_DIR}
 )
-
-# Libraries
-set(LIBS
-    CBot
-    colobotbase # Needed for error strings (TODO: why are they on Colobot side? :/)
```

Comment:
you removed the comment :<

Reply:
Ah, sorry, will bring it back :D I'm not sure why I did this, probably in the hurry I accidentally removed it without thinking.